.github/scripts/set_benchmark_param.sh

```diff
@@ -12,15 +12,24 @@ else
     echo "MODEL_FORMAT=" >> "$GITHUB_ENV"
 fi
 
-if [[ $1 == *"llama"* ]] || [[ $1 == *"Llama"* ]]
+if [[ $1 == *"llama2"* ]] || [[ $1 == *"Llama-2"* ]]
 then
     echo "MAX_ENTRY_COUNT=--cache-max-entry-count 0.95" >> "$GITHUB_ENV"
+
 else
     echo "MAX_ENTRY_COUNT=--cache-max-entry-count 0.90" >> "$GITHUB_ENV"
 fi
 
+if [[ $1 == *"llama2"* ]] || [[ $1 == *"Llama-2"* ]]
+then
+    echo "BATCHES=128" >> "$GITHUB_ENV"
```

Comment:
we recommend batch size 128 for llama2-13b. The rest llama models use batch size 256